database/migrations/platform/2024_05_10_000000_drop_leaderboardentry_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::dropIfExists('LeaderboardEntry');
```

Comment:
There's still some references to this table in `getLeaderboardsForGame`.

Reply:
Thanks. I believe this may actually turn out to be the most impactful change in the PR.

It appears `getLeaderboardsForGame()` is only being used to populate a count value called `$numLeaderboards` on game pages. In latest I've simply refactored this to `$gameModel->leaderboards()->visible()->count()` and deleted the function.

The SQL query in `getLeaderboardsForGame()` was accounting for _half the loading performance_ of several game pages on my local, clocking in at roughly 100-300ms. The new Eloquent query takes less than 1ms, a tremendous improvement.